crypto/rand/rand_unix.c

```diff
@@ -264,7 +254,7 @@ static ssize_t syscall_random(void *buf, size_t buflen)
      * Note: 'buflen' equals the size of the buffer which is used by the
      * get_entropy() callback of the RAND_DRBG. It is roughly bounded by
      *
-     *   2 * RAND_POOL_FACTOR * (RAND_DRBG_STRENGTH / 8) = 2^14
+     *   2 * DRBG_MINMAX_FACTOR * (RAND_DRBG_STRENGTH / 8) = 2^13
```

Comment:
could you please explain this change.

Reply:
Could be an aftermath from https://github.com/openssl/openssl/pull/8023#issuecomment-454603136. (But I didn't check it.)